modules/change_management/languages/en.json

```diff
@@ -5,6 +5,8 @@
             "PLURAL_NAME": "Change Management"
         },
         "FIELDS": {
+            "COMMITSTATUS": "Commit Status",
```

Comment:
This change also needs to be added to the French, Korean, and Chinese language files.

Reply:
done